build-system/tasks/ava.js

```diff
@@ -15,29 +15,30 @@
  */
 'use strict';
 
-const gulp = require('gulp');
-const gulpAva = require('gulp-ava');
+const {exec} = require('../common/exec');
 
 /**
  * Runs ava tests.
- * @return {!Vinyl}
+ * @param {function} cb
  */
-function ava() {
-  return gulp
-    .src([
-      require.resolve('./get-zindex/test.js'),
-      require.resolve('./prepend-global/test.js'),
-    ])
-    .pipe(
-      gulpAva({
-        'concurrency': 5,
-        'failFast': true,
-      })
-    );
+function ava(cb) {
```

Comment:
could we turn this into either a Promise like api or async, or is the node callback pattern the only way here?

Reply:
Good catch! Made this `async`, and signaled success / failure using `execOrDie`. Once we remove the `gulp` wrapper (soon, I hope), this should work as a simple node script.